node/service/src/lazy_loading/mod.rs

```diff
@@ -571,6 +573,14 @@ where
 							maybe_current_para_head?.encode(),
 						));
 
+						// Get the mocked timestamp
+						let mut timestamp = 0u64;
+						TIMESTAMP.with(|x| {
+							timestamp = x.clone().take() + RELAY_CHAIN_SLOT_DURATION_MILLIS;
```

Comment:
useless clone, and bad usage of take, anyway please migrate to AtomicU64 and you will don't that

Reply:
This was copied over from Tanssi as our use of time was breaking the mocks. Will fix that, though.